### DISCUSSION TITLE
Try in pingcap/cloud-assets-utils

### DISCUSSION THREAD

**Discussion by g1eny0ung** [actions-ml/setup-ocaml#2] [Show and tell] [answered]
From https://github.com/avsm/setup-ocaml/issues/4#issuecomment-752323604, I have been already tested this action in https://github.com/pingcap/cloud-assets-utils.

@smorimoto Seems everything is ok. The setup speed is significantly faster than https://github.com/avsm/setup-ocaml.

Here is the test workflow link: https://github.com/pingcap/cloud-assets-utils/actions/runs/452090724.

Thx for making this action. 😆

**Comment by smorimoto** [comment, marked as answer]
@g1eny0ung Thanks for trying this!

**Reply by shonfeder** [reply]
I've just tried it too, and the setup took a 1:09 the first time. Nice work and thanks!

**Reply by smorimoto** [reply]
I haven't officially released it yet, so it's quite unstable, but I'm glad to know that you have a good impression :)

**Reply by shonfeder** [reply]
Noted! I can report back if anything goes amiss, if that would be helpful.

**Reply by smorimoto** [reply]
Thank you!

**Comment by smorimoto** [comment]
See https://github.com/actions-ml/setup-ocaml/discussions/4#discussioncomment-545075
